src/octoprint/util/gcodeInterpreter.py

```diff
@@ -122,13 +130,15 @@ def _load(self, gcodeFile, printer_profile, throttle=None):
 					z = getCodeFloat(line, 'Z')
 					e = getCodeFloat(line, 'E')
 					f = getCodeFloat(line, 'F')
-					#Check if x,y or z exceeds printing area
-					width=float(printer_profile["volume"]["width"])
-					depth=float(printer_profile["volume"]["depth"])
-					height=float(printer_profile["volume"]["height"])
-					if not self.warn and (x>width or y>depth or z>height):
-						self.warn=True 
-						self._logger.warn("Object is bigger than the printing area")
+					#Add value to list in case move includes extrusion
+					if e is not None and (e-previousE)>0:
+						if x is not None:
+							xList.append(x)
```

Comment:
Good to know. I did something similar in a first version but I thought this would be done more slow and spending more memory.